source/localizable/tutorial/installing-addons.md

```diff
@@ -18,7 +18,11 @@ Run the following command to install the addon:
 ember install ember-cli-tutorial-style
 ```
 
-Starting the server will incorporate the new CSS and refreshing our browser window will give you this:
+Since Ember addons are regular npm packages, `ember install` installs them in
```

Comment:
@toddjordan I intended regular to mean installable via `npm install`, but, sure, if that's not coming across I'll make the changes.